lib/chef/handler/sns/config.rb

```diff
@@ -209,6 +209,23 @@ def topic_arn(arg = nil)
         end
 
         #
+        # Gets or sets MessageStructure SNS request parameter.
+        #
+        # @param arg [String] MessageStructure.
+        #
+        # @return [String] MessageStructure.
+        #
+        # @api public
+        #
+        def message_structure(arg = nil)
+          set_or_return(
+            :message_structure,
+            arg,
+            kind_of: String
+          )
+        end
```

Comment:
Could you add some tests [here](https://github.com/zuazo/chef-handler-sns/blob/master/test/test_chef_handler_sns_config.rb#L32) for the `message_structure` method?

Please, skim-read the already existing tests. You should not have trouble understanding them even if you have never done tests before with [minitest](http://docs.seattlerb.org/minitest/). They are very verbose but it does not take long to read them once you have understood one.

Of course, do not hesitate to consult me if you have any doubts.